docs/GetStarted/getting_started_python.md

```diff
@@ -33,7 +33,14 @@ $ python3 -m pip install tf-nightly
 
 ## Running Python Tests
 
-To run tests for core Python bindings:
+To run tests for core Python bindings build with CMake:
```

Comment:
nit: "built" (here and below). I can fix as part of merging.

Reply:
Skipping 'ready to pull' here since I'm manually merging instead of using the automation, this is just waiting on someone else to approve on our side to merge.